datafusion/sqllogictest/test_files/scalar.slt

```diff
@@ -1775,7 +1775,7 @@ CREATE TABLE test(
 (-14, -14, -14.5, -14.5),
 (NULL, NULL, NULL, NULL);
 
-query IIRRIR rowsort
+query IRRRIR rowsort
```

Comment:
This is a minor difference; previously we were able to cast floats to integer, but with the new signature we fallback to computing as float if either base or exponent is float.